CHANGELOG.md

```diff
@@ -2,6 +2,33 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.14.0](https://github.com/aws/aws-cdk/compare/v1.13.1...v1.14.0) (2019-10-22)
+
+
```

Comment:
Add prominent note about the imminent demise of node 8 please !!

Reply:
Oops forget. Thanks for reminding me

Reply:
Ok, updated. Please take another look and make any edits you see fit.